app/src/main/java/xyz/jhughes/laundry/activities/MachineActivity.java

```diff
@@ -145,6 +146,10 @@ public boolean onCreateOptionsMenu(Menu menu) {
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
+            case android.R.id.home:
+                    Intent i = getParentActivityIntent().putExtra("forceMainMenu", true);
+                    navigateUpTo(i);
```

Comment:
The back button works for me in the current app? Is there a bug I don't know about?